# json of termination configs
string cfg_json